views/shifts.pug

```diff
@@ -3,6 +3,8 @@ extends layout
 block extrahead
   link(rel="stylesheet" href="/stylesheets/shifts.css")
   script(src="/javascripts/shifts.js" type="text/javascript")
+  script(src="/javascripts/default_scripts.js" type="text/javascript")
```

Comment:
stupid question but why do you need this since you did not add anything to defaults_scripts?

Reply:
No idea, actually.